packages/flutter_tools/test/data/intellij/plugins/META-INF/plugin.xml

```diff
@@ -0,0 +1,93 @@
+<idea-plugin version="2">
```

Comment:
Good catch. This is left over from constructing the *.jar file and should not be there. Now removed.